incubator/virtualcluster/pkg/controller/virtualcluster/virtualcluster_controller.go

```diff
@@ -185,28 +185,88 @@ func (r *ReconcileVirtualcluster) createPKI(vc *tenancyv1alpha1.Virtualcluster,
 	return nil
 }
 
+// genInitialClusterArgs generates the values for `--inital-cluster` option of etcd based on the number of
+// replicas specified in etcd StatefulSet
+func genInitialClusterArgs(replicas int32, stsName, svcName string) (argsVal string) {
+	for i := int32(0); i < replicas; i++ {
+		peerAddr := fmt.Sprintf("%s-%d=https://%s-%d.%s:2380", stsName, i, stsName, i, svcName)
```

Comment:
I am confused about the `2380` here

Reply:
2380 is the default port for etcd peer to communicate with each other.

Reply:
assign the default port number to a constant